src/routes/AppRoutes.jsx

```diff
@@ -0,0 +1,29 @@
+import { Route, Routes } from "react-router-dom";
+import { UserAuth } from "../context/AuthContext";
+import { authRoutes } from "./AuthRoutes";
+import { pageRoutes } from "./StaticRoutes";
+import { protectedRoutes } from "./DashboardRoutes";
+
+export const RenderRoutes = () => {
```

Comment:
This is one of the big changes I have made. 
This component is in charge of adding all of the routes specified in the various routes folders into one list. 

 It actually reads/works in the following flow:

On line 25, I create an array object. This object consists of all the entries from any of our separate route files and collates them into one list..

Now having that array at the ready, we then iterate through each entry of the array (Line 12) and map the output accordingly with one of the following scenarios:

- If the route is an Authenticated route (r.isAuthenticated) and the user is authenticated (!user.IsAnonymous) then return a route 
- Else if the route ins't an authenticated route, then just return it
- Else return false (This is a dead end line and serves as a JUST IN CASE)

Reply:
Way too smart for me